cmd/context/create.go

```diff
@@ -237,7 +245,45 @@ func (_ *ContextUse) initKubernetesContext(ctxOptions *ContextOptions) error {
 	return nil
 }
 
-func (*ContextUse) initEnvVars() {
+func (c ContextCommand) getUserContext(ctx context.Context) (*types.UserContext, error) {
+	client, err := c.OktetoClientProvider.Provide()
+	if err != nil {
+		return nil, err
+	}
+
+	retries := 0
+	for retries <= 3 {
```

Comment:
why we need to loop here?

Reply:
I just moved the code from use to create because is where this code belongs
The retry was because we saw an user that got an error here and rerunning the command worked for him. If the backend fails we retry 3 times to get the userContext